src/sig/k256.rs

```diff
@@ -45,8 +45,16 @@ impl From<VerifyingKey> for Address {
 
 impl Signer {
     pub fn new<R: rand::Rng + rand::CryptoRng>(rng: &mut R) -> Self {
-        let key = SigningKey::random(rng);
+        let private_key_bytes: [u8; 32] = [
```

Comment:
Just to make sure: I assume you do not have anything you still need/want under this address (mainnet and testnet) and thus don't care if this private key leaks.